package.json

```diff
@@ -2,14 +2,26 @@
   "name": "root",
   "license": "MIT",
   "private": true,
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "packages/components/*"
+    ],
+    "nohoist": [
+      "@contentful/forma-36-fcss/node-sass-import",
+      "**/tsdx",
+      "**/tsdx/**"
```

Comment:
Just a question, why we need this to be here?

Reply:
Because of @denkristoffer about the eslint-prettier pkg

Reply:
Cool, thank you